gustaf/spline/ffd.py

```diff
@@ -397,8 +416,8 @@ def show(self, **kwargs) -> Any:
             Returns, if applicable, the vedo plotter. 'close=False' as argument
             to get the plotter.
         """
-        if not (self._spline is None or self._mesh is None):
-            raise ValueError("Please set a spline before using this function.")
+        if self._spline is None and self._mesh is None:
+            raise ValueError("Please set a mesh before calling show()")
```

Comment:
Previous one was causing error in `examples/show_ffd.py`.  
strictly, you only need to set mesh as default spline will be given. on this note, would you like to add a feature, "padding" in ffd, to avoid some floating number issues?